buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -1,27 +1,34 @@
 
 object Vers {
     //Plugins
-    const val dokkav = "0.9.18"
-    const val gradle_release_plugin = "1.3.9"
-    const val asciidoctor = "1.5.9.2"
+    const val dokkav = "0.10.1"
+    const val gradle_release_plugin = "1.3.17"
+    const val asciidoctor = "1.6.0"
+    const val asciidoctor_convert = "2.4.0"
+    const val nexus_staging_plugin = "0.21.2"
+    const val nexus_publish_plugin = "0.4.0"
 
     //Dependencies
-    const val kotlin = "1.3.50"
-    const val jfix_stdlib = "1.0.59"
-    const val lombok = "1.18.4"
-    const val jackson = "2.9.6"
-    const val jackson_kotlin = "2.9.6"
-    const val apache_curator = "4.2.0"
-    const val logback = "1.1.11"
-    const val commons_io = "2.4"
-    const val junit4 = "4.12"
+    const val kotlin = "1.3.70"
+    const val lombok = "1.18.12"
+    const val jackson = "2.10.3"
+    const val jackson_kotlin = "2.10.3"
+    const val apache_curator = "4.3.0"
+    const val logback = "1.2.3"
+    const val commons_io = "2.6"
+    const val junit_jupiter = "5.6.2"
+
+    const val aggregating_profiler = "1.5.16"
+    const val jfix_stdlib = "3.0.0"
+    const val jfix_dynamic_property = "2.0.3"
```

Comment:
Yes, profiler and dynamic property for lock manager,  stdlib for zk testing server

Reply:
Could components, which use these dependencies optional? Default use generic thread without profiler/dynamic_property, otherwise use provided. In my opinion these dependencies are redundant.